.travis.yml

```diff
@@ -19,7 +19,7 @@ install:
 # Assist with ci test debugging:
 #   - DEBUG=1
 before_script:
-    - image="kylemanna/openvpn"
+    - image="r0p0s3c/openvpn"
```

Comment:
And un-necessary, this builds the image, it doesn't need to pull it, so the name is largely irrelevant.